api/tests/test_winery.py

```diff
@@ -56,7 +57,7 @@ def test_winery_endpoint_create_should_not_be_allowed(self):
             reverse('winery-list'),
             data
         )
-        self.assertEqual(status.HTTP_405_METHOD_NOT_ALLOWED, response.status_code)
+        self.assertEqual(status.HTTP_401_UNAUTHORIZED, response.status_code)
```

Comment:
The permission is using the user winery to grant access since there's no winery yet, always gives an unauthorized message, If we want it to be different we should change the permission.